orion/sites/admin/src/index.js

```diff
@@ -22,6 +23,8 @@ const Home = () => <HomePage/>;
 const Programs = () => <ProgramPage/>;
 const ProgramEdit = () => <ProgramEditPage/>;
 const ProgramEditMatch = ({match}) => <ProgramEditPage programId={match.params.programId}/>;
+const AchieveEdit = () => <AchieveEditPage/>;
+const AchieveEditMatch = ({match}) => <AchieveEditPage year={match.params.year}/>;
```

Comment:
Can you place these 2 next to `Achieve` on line 19?
So it should look like:
```
const Achieve = () => <AchievePage/>;
const AchieveEdit = () => <AchieveEditPage/>;
const AchieveEditMatch = ({match}) => <AchieveEditPage year={match.params.year}/>;
```
We want to keep related things next to each other.